test/components/DateInput_spec.jsx

```diff
@@ -69,6 +69,47 @@ describe('DateInput', () => {
       );
     });
 
+    describe('screen reader message', () => {
+      const id = 'date';
+      const screenReaderMessage = 'My screen reader message';
+      const screenReaderMessageName = `DateInput__screen-reader-message-${id}`;
+      const screenReaderMessageId = `#${screenReaderMessageName}`;
```

Comment:
Yeah your correct, I've updated the names of these variables. 

I also moved the creation of the wrappers into a before loop since they were all the same, which reduces the duplication and makes it easier to read.